### PR TITLE
initial registries setup and validation v2.7

### PR DESCRIPTION
**Pull request by izaac**
Forward port of: https://github.com/rancher/rancher/pull/40930

### CODE REVIEW

**Review by igomez06 — DISMISSED** [review]
Looks good

File: tests/v2/validation/Jenkinsfile.e2e [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    def golangTestContainer = "${job_name}${env.BUILD_NUMBER}-golangtest"
+    def buildTestContainer = "${job_name}${env.BUILD_NUMBER}-buildtest"
+    def cleanupTestContainer = "${job_name}${env.BUILD_NUMBER}-cleanuptest"
+    def golangImageName = "rancher-registries-validation-${job_name}${env.BUILD_NUMBER}"
+    def registriesVolume = "RegitriesSharedVolume-${job_name}${env.BUILD_NUMBER}"
+    def testsDir = "/root/src/github.com/rancher/rancher/tests/v2/validation/${env.TEST_PACKAGE}"
+    def testResultsOut = "results.xml"
+    def envFile = ".env"
+    def rancherConfig = "rancher_env.config"
+    def branch = "release/v2.6"
```
Comment (caliskanugur) [inline tests/v2/validation/Jenkinsfile.e2e:19]: I've got just a small nit here to default this 2.7.

**Review by caliskanugur — COMMENTED** [review]
I've got a small nit about the default branch since this is gonna merge to 2.7. Looks good to me.

**Review by caliskanugur — APPROVED** [review]
Looks good to me

**Review by igomez06 — APPROVED** [review]

**Pull request merged**